src/app/modules/group/components/pending-request/pending-request.component.html

```diff
@@ -1,5 +1,5 @@
 <alg-section-paragraph
-  *ngIf="requests.length"
+  *ngIf="requests.length || isPublic"
```

Comment:
That would clearly be better. Also wouldn't it make sense to also load the pending-request when `group.isCurrentUserManager` ?

Reply:
" to also load" -> you meant "to only load" ? Yes exactly! I don't know why it has not been done though.

Reply:
I thought that in a"public group any member could add other member while in private ones only manager would accept people.

If that's not the case, then I guess in a users can't make requests to join a private group so it doesn't make sense to show the pending requests ?

Anyway, I'm going to add `group.isCurrentUserManager` to the condition.

Reply:
The reference to know about attributes is the db doc: https://france-ioi.github.io/algorea-devdoc/dbschema/ (in `groups` table in this case). Public means "Whether it is visible to all users (through search) and open to join requests" so there is no join requests for such groups which are not public. Actually "is_open" is quite related as well.
But the main problem with not fetching anything with is_public and is_open is if you change the property of group when there are requests which are pending, e.g. but you could "is_open <-- false" manually when you do not want more registrations, but you may still have pending request to validate, so not checking is dangerous. 
So to prevent any problem... don't check for "is_public". But the manager check of course is useful.

Reply:
fixed